docs/REST API/rest-api.md

```diff
@@ -5525,11 +5526,13 @@ List all configuration profiles for macOS and Windows hosts enrolled to Fleet's
       "checksum": "aCLemVr)",
       "labels_include_all": [
         {
-          "name": "Label name 1",
-          "broken": true
+          "name": "Label name 2",
+          "broken": true,
+          "id": 2
```

Comment:
Nit: a "broken" label does not have an ID (because it means it has been deleted).

Reply:
Yes! Thanks Martin. Opened a PR here: https://github.com/fleetdm/fleet/pull/21263